src/config.ts

```diff
@@ -16,7 +16,7 @@ import utils from './utils'
 
 export const defaultConfig = {
   type: DEFAULT_CRYPTOSYSTEM,
-  curve: DEFAULT_EccCurve,
+  curve: DEFAULT_ECC_CURVE,
```

Comment:
I was about to ask why you changed these from screaming snake case to camel case. Then realized it was the other way around and I had them as camel case... anyways, good looks thanks for changing this! :+1:

Reply:
also changed `DEFAULT_RSA_SIZE` as well

Reply:
Thanks, weird that eslint didn't complain about this one 🤷